project-minimization-plugin/src/test/kotlin/SettingsUITest.kt

```diff
@@ -75,4 +75,10 @@ class SettingsUITest : BasePlatformTestCase() {
         component.temporaryProjectLocation = "new-location"
         assertEquals("new-location", component.temporaryProjectLocation)
     }
+
+    fun testIgnoreList() {
+        component.ignorePaths = listOf("/Users/Roman.Vsemirnov/Desktop/project-minimization/project-minimization-plugin/build.gradle.kts")
+        assertEquals(1, component.ignorePaths.size)
+        assertEquals("/Users/Roman.Vsemirnov/Desktop/project-minimization/project-minimization-plugin/build.gradle.kts", component.ignorePaths[0])
+    }
```

Comment:
Tests shouldn't contain local files from your computer :)

Reply:
oops, forgot to fix that and make normal test